triedb/pathdb/asyncnodebuffer.go

```diff
@@ -226,7 +226,8 @@ func (nc *nodecache) node(owner common.Hash, path []byte, hash common.Hash) (*tr
 	}
 	if n.Hash != hash {
 		dirtyFalseMeter.Mark(1)
-		log.Error("Unexpected trie node in async node buffer", "owner", owner, "path", path, "expect", hash, "got", n.Hash)
+		// difflayer cache miss maybe hit this, which is normal case, and the caller can retry it.
+		log.Debug("Unexpected trie node in async node buffer", "owner", owner, "path", path, "expect", hash, "got", n.Hash)
```

Comment:
If the data is corrupted, there is no `Unexpected trie node` log. Why not change it to `WARN`?

Reply:
Updated.